ClassFieldTheory/GroupCohomology/_9_CyclicGroup.lean

```diff
@@ -41,7 +43,24 @@ noncomputable section
 -/
 abbrev gen : G := IsCyclic.exists_generator.choose
 
-variable {G}
+variable {G} in
+lemma gen_generate (x : G) : x ∈ Subgroup.zpowers (gen G) :=
+  IsCyclic.exists_generator.choose_spec x
```

Comment:
Do we need this lemma when we can just use `IsCyclic.exists_generator.choose_spec`?